docs/deploy_features.md

```diff
@@ -17,35 +17,52 @@ You should typically enable these features before running `azd up`. Once you've
 * [Deploying with private endpoints](#deploying-with-private-endpoints)
 * [Using local parsers](#using-local-parsers)
 
-## Using GPT-4
+## Using GPT-4 or GPT-4o
 
-We generally find that most developers are able to get high quality answers using GPT 3.5. However, if you want to try GPT-4, you can do so by following these steps:
+We generally find that most developers are able to get high quality answers using GPT 3.5. However, if you want to try GPT-4 or GPT-4o, you can do so by following these steps:
 
 Execute the following commands inside your terminal:
 
 1. To set the name of the deployment, run this command with a new unique name.
 
+    For GPT-4:
     ```bash
     azd env set AZURE_OPENAI_CHATGPT_DEPLOYMENT chat4
     ```
+    For GPT-4o:
```

Comment:
4o doesnt technically need a different name, this kind of implies it does. Can we just say "You can use any deployment name, as long as it's unique in your Azure OpenAI account."